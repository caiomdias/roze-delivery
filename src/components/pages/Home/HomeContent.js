import React from 'react';
import Search from '../../common/search/Search'

const HomeContent = () => (
	<div className="home-content">
		<div className="container">
			<div className="content-wrapper">
				<h1>
					<strong>Bebidas geladas </strong> a <strong>preço </strong>de mercado na sua casa <strong>agora</strong>
				</h1>
				<Search
					type="address"
					placeholder="Insera seu endereço para ver preço"
				/>
			</div>
		</div>
	</div>
);

export default HomeContent;
