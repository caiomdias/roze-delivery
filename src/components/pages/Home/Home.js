import React from 'react';
import HomeContent from './HomeContent';
import whiteLogo from '../../../assets/images/png/logo.png';
import Header from '../../common/Header';

const headerConfig = {
	image: whiteLogo,
	hasSearch: false,
	style: {
		width: '50px',
		cursor: 'unset',
	}
}

const Home = () => (
	<>
		<Header config={headerConfig} />
		<HomeContent />
	</>
);

export default Home;
