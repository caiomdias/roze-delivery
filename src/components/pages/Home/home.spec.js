import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

jest.mock('react-router-dom', () => ({
    useLocation: jest.fn().mockReturnValue({
      pathname: '/',
      search: '',
      hash: '',
      state: null,
      key: '5nvxpbdafa',
	}),
	useHistory: jest.fn().mockReturnValue({
		push: jest.fn().mockReturnValue()
	})
}));

describe("Home Component", () => {
	it('render correctly Home component', () => {  
		const homeComponent = renderer.create(<Home />);
		expect(homeComponent).toMatchSnapshot();
	});
});