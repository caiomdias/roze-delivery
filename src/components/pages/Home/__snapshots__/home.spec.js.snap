// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component render correctly Home component 1`] = `
Array [
  <header
    className="header"
  >
    <div
      className="container content"
    >
      <div
        className="header__logo"
        onClick={[Function]}
      >
        <img
          alt="Jose Entregas"
          src="test-file-stub"
          style={
            Object {
              "cursor": "unset",
              "width": "50px",
            }
          }
        />
      </div>
    </div>
  </header>,
  <div
    className="home-content"
  >
    <div
      className="container"
    >
      <div
        className="content-wrapper"
      >
        <h1>
          <strong>
            Bebidas geladas 
          </strong>
           a 
          <strong>
            preço 
          </strong>
          de mercado na sua casa 
          <strong>
            agora
          </strong>
        </h1>
        <div
          className="search"
        >
          <div
            className="search__input"
          >
            <div
              className="icon"
            />
            <input
              className="input"
              onChange={[Function]}
              placeholder="Insera seu endereço para ver preço"
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
